molecule/check_facts/hosts.yml

```diff
@@ -0,0 +1,47 @@
+---
+cluster:
+  vars:
+    # common connection opts
+    ansible_user: root
+    ansible_connection: docker
+    become: true
+    become_user: root
+
+    # common cartridge opts
+    cartridge_app_name: myapp
+    cartridge_cluster_cookie: secret-cookie
+
+    cartridge_custom_steps_dir: './tasks'
+    cartridge_custom_scenarios:
+      custom_scenario:
+        - custom_task
+
+  hosts:
```

Comment:
Seems that all this hosts are redundant except one. The same for replicaset group.